modules/examples/todolist-service/src/main/scala/todo/model/model.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2017-2018 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package todo
```

Comment:
Should we align the namings in the example packages between repositories?

For example, in `freestyle-rpc` we are using this naming https://github.com/frees-io/freestyle-rpc/pull/223/files#diff-daf5628654bc401364d5e548c18e74efR17

IMO we should use this naming, `package examples.todolist.service`. Thoughts?

Reply:
I totally agree. I didn't know, but I think it's a good idea to homogenize this. :100: